Middleware/ARM/ota-pal-psa/License.md

```diff
@@ -0,0 +1,21 @@
+The software is provided under MIT License.
```

Comment:
ota-pal-psa should be incorporated as a submodule so we can easily contribute changes upstream.